relay-server/src/services/buffer/envelope_stack/file_backed.rs

```diff
@@ -506,38 +485,6 @@ mod tests {
         );
     }
 
-    #[tokio::test]
-    async fn test_malformed_data() {
```

Comment:
@iambriccardo I removed this test because it looked like it tested the same thing as `test_incomplete_envelope`. Let me know if I missed something.

Reply:
Now with your changes, they test the same code path, so it's fine to delete it, thanks!